api/config/v1/imex.go

```diff
@@ -0,0 +1,98 @@
+/**
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+**/
+
+package v1
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math"
+)
+
+const (
+	ImexChannelEnvvar = "NVIDIA_IMEX_CHANNELS"
```

Comment:
nit: `EnvVar`

Reply:
The use of `Envvar` is consistent with other such variables in the code base. I would rather create a chore to replace all occurrences and update this then.

Reply:
- https://github.com/NVIDIA/k8s-device-plugin/pull/988